main.py

```diff
@@ -2,6 +2,68 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+
+    actors = (
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    )
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for actor in actors:
+        Actor.objects.create(
```

Comment:
You can unpack lists in place, if you want. For example,
for value1, value2 in list_of_pairs: